src/frontend/komponenter/Fagsak/Behandlingsresultat/OvergangsordningAndelRad.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { useState } from 'react';
+
+import deepEqual from 'deep-equal';
+import styled from 'styled-components';
+
+import { Table } from '@navikt/ds-react';
+
+import OvergangsordningAndelSkjema from './OvergangsordningAndelSkjema';
+import { useOvergangsordningAndel } from '../../../context/OvergangsordningAndelContext';
+import type { IBehandling } from '../../../typer/behandling';
+import type { IRestOvergangsordningAndel } from '../../../typer/overgangsordningAndel';
+import { Datoformat, isoMånedPeriodeTilFormatertString } from '../../../utils/dato';
+import { lagPersonLabel } from '../../../utils/formatter';
+
+interface IOvergangsordningRadProps {
+    overgangsordningAndel: IRestOvergangsordningAndel;
+    åpenBehandling: IBehandling;
+}
+
+const PersonCelle = styled.div`
+    display: flex;
+    svg {
+        margin-right: 1rem;
+    }
+`;
+
+const OvergangsordningAndelRad: React.FunctionComponent<IOvergangsordningRadProps> = ({
+    overgangsordningAndel,
+    åpenBehandling,
+}) => {
+    const [åpenOvergangsordningAndel, settÅpenOvergangsordningAndel] = useState<boolean>(
+        overgangsordningAndel.personIdent === null
+    );
+
+    const { hentSkjemaData } = useOvergangsordningAndel();
+
+    const erSkjemaForandret = () =>
+        !deepEqual(
+            {
+                ...overgangsordningAndel,
+                antallTimer:
+                    typeof overgangsordningAndel.antallTimer === 'number'
+                        ? overgangsordningAndel.antallTimer
+                        : 0,
```

Comment:
Fungerer dette som tiltenkt? Her sørger vi for at `antallTimer` blir et `Number`, men hentSkjemaData() returnerer `undefined` for `antallTimer` når feltet ikke er satt i skjemaet. Ville det kanskje fungert å bare fjerne hele overstyringen av `antallTimer` her? 🤔 Mulig det er noe jeg ikke ser her også 🙈

Reply:
Mistenker altså at `erSkjemaForandret` vil returnere true selv om skjemaet ikke er forandret, så lenge `antallTimer` ikke er satt i skjemaet.